doc/classes/AnimationRootNode.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		[AnimationRootNode] is a base class for [AnimationNode]s that hold a complete animation. A complete animation refers to the output of an [AnimationNodeOutput] in an [AnimationNodeBlendTree] or the output of another [AnimationRootNode]. Used for [member AnimationTree.tree_root] or in other [AnimationRootNode]s.
+		Examples of built-in root nodes include [AnimationNodeBlendTree], which allows blending [AnimationNode]s between each other using operations such as mix, blend2, blend3, one shot, etc. There is also [AnimationNodeStateMachine], where all [AnimationNode]s you insert are states and you define which are playing, how they are blending as well as how they should transition to each other. Furthermore, an [AnimationNodeBlendSpace2D] allows linear blending between [b]three[/b] [AnimationNode]s. Whereas [AnimationNodeBlendSpace1D] allows linear blending only between [b]two[/b] [AnimationNode]s.
```

Comment:
```suggestion
		Examples of built-in root nodes include [AnimationNodeBlendTree] (which allows blending [AnimationNode]s between each other using operations such as mix, blend2, blend3, one shot, etc.) and [AnimationNodeStateMachine], where all [AnimationNode]s you insert are states and you define which are playing, how they are blending as well as how they should transition to each other. Furthermore, an [AnimationNodeBlendSpace2D] allows linear blending between [b]three[/b] [AnimationNode]s, whereas [AnimationNodeBlendSpace1D] allows linear blending only between [b]two[/b] [AnimationNode]s.
```

Reply:
Hm, I wanted to avoid too long sentences to make it easier to digest what info belongs together. Therefore I'm not too fond of the `(…) and `.  
But I like your `, whereas` change. Makes it feel a bit more fluent.

How do you feel about committing the latter change suggestion and keeping the former as-is?

Reply:
@MJacred Sure! Go ahead :)

Reply:
I agree that this part shouldn't be too verbose. Which is why in my original suggestion I implied it would be better to simplify what each node does so it fits under parenthesis. If you can adjust the style this way, that would be a great improvement!

> Examples of built-in root nodes include [AnimationNodeBlendTree] (allows blending nodes between each other using various modes), ...

Reply:
You mean like this? (using fake rendering)

Examples of built-in root nodes include [AnimationNodeBlendTree](url) (which allows blending [AnimationNode](url)s between each other using operations such as mix, blend2, blend3, one shot, etc.), [AnimationNodeStateMachine](url) (where all [AnimationNode](url)s you insert are states and you define which are playing, how they are blending as well as how they should transition to each other), [AnimationNodeBlendSpace2D](url) (allows linear blending between **three** [AnimationNode](url)s), [AnimationNodeBlendSpace1D](url) (allows linear blending only between **two** [AnimationNode](url)s).

At this point I'd say it's only a difference of writing style. But if you insist, I'll use parenthesis.

Reply:
> Yes, like that, but I would reduce the first two descriptions further

Done, thanks for sticking with me : )